### PR TITLE
Refactor mf

### PR DESCRIPTION
**Pull request by foleymichelle9**
Refactored my methods and cleaned up indentations + spaces

### CODE REVIEW

File: lib/game_team.rb [hunk 1]
```diff
@@ -107,47 +107,32 @@ def self.gets_team_shots_goals_count(season)
     stats_by_team = get_goal_shots_by_game_team(matches)
   end
 
-#Michelle Start
```
Comment (foleymichelle9) [inline lib/game_team.rb:110]: Woot woot combined these two!

**Comment by Rostammahabadi** [comment]
looks good!

File: lib/game_team.rb [hunk 2]
```diff
@@ -107,47 +107,32 @@ def self.gets_team_shots_goals_count(season)
...
-        tackles_by_team[key] = total_tackles
-      end
+    tackles_by_team = {}
+    games_by_team_name(season_id).each do |key, value|
+      total_tackles = value.sum { |value| value.tackles}
+      tackles_by_team[key] = total_tackles
+    end
       tackles_by_team
   end
-#####45 Seconds
```
Comment (foleymichelle9) [inline lib/game_team.rb:139]: I believe this is no longer accurate so I deleted it

File: test/game_team_test.rb [hunk 3]
```diff
@@ -168,18 +168,9 @@ def test_most_accurate_team
 
     assert_equal "6", GameTeam.most_accurate_team("20122013")
   end
-#Michelle Start
-  def test_it_can_find_game_ids_by_season
```
Comment (foleymichelle9) [inline test/game_team_test.rb:172]: unnecessary tests

File: lib/game_team.rb [hunk 4]
```diff
@@ -107,47 +107,32 @@ def self.gets_team_shots_goals_count(season)
...
-      tackles_by_team = {}
-      games_by_team_name(season_id).each do |key, value|
-        total_tackles = value.sum { |value| value.tackles}
-        tackles_by_team[key] = total_tackles
-      end
+    tackles_by_team = {}
+    games_by_team_name(season_id).each do |key, value|
+      total_tackles = value.sum { |value| value.tackles}
+      tackles_by_team[key] = total_tackles
+    end
```
Comment (stellakunzang) [inline lib/game_team.rb:122]: yesssssss! (big fan of proper indentation)

File: lib/game_team.rb (hunk 1 above)
Comment (perryr16) [reply lib/game_team.rb:110]: speedy code

File: lib/game_team.rb (hunk 2 above)
Comment (perryr16) [reply lib/game_team.rb:139]: aaaawww yeah

**Review by perryr16 — APPROVED** [review]
approved-rp

**Comment by stellakunzang** [comment]
Nailed it!

**Pull request merged**